src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -13,13 +13,9 @@
 
     @Override
     public List<Machine> getAll(Class<? extends Machine> type) {
-        if (type == Bulldozer.class) {
-            return new ArrayList<>(new BulldozerProducer().get());
-        } else if (type == Excavator.class) {
-            return new ArrayList<>(new ExcavatorProducer().get());
-        } else {
-            return new ArrayList<>(new TruckProducer().get());
-        }
+        return type == Bulldozer.class ? new ArrayList<>(new BulldozerProducer().get()) :
+                type == Excavator.class ? new ArrayList<>(new ExcavatorProducer().get()) :
+                        new ArrayList<>(new TruckProducer().get());
```

Comment:
wow. amazing, but hard to read)) https://mate-academy.github.io/jv-program-common-mistakes/java-core/pecs/pecs.html here is a hint of what I meant